src/compatibility.ts

```diff
@@ -58,19 +58,22 @@ const deprecatedOrgAliases = {
  */
 export const optionalOrgFlagWithDeprecations = optionalOrgFlag({
   ...deprecatedOrgAliases,
+  description: messages.getMessage('flags.targetOrg.description'),
 });
 
 /**
  * @deprecated
  */
 export const requiredOrgFlagWithDeprecations = requiredOrgFlag({
   ...deprecatedOrgAliases,
+  description: messages.getMessage('flags.targetOrg.description'),
 });
 
 /**
  * @deprecated
  */
 export const requiredHubFlagWithDeprecations = requiredHubFlag({
-  aliases: ['targetdevhubusername', 'v'],
```

Comment:
I was testing the hub flag and noticed `-v` was deprecated, but the sf does set the char to `v` so it might confuse users. We only want to deprecate `--targetdevhubusername`, `--target-org/-o/-v` should be valid.

https://github.com/salesforcecli/sf-plugins-core/blob/main/src/flags/orgFlags.ts#L128

![Screen Shot 2022-11-17 at 15 37 09](https://user-images.githubusercontent.com/6853656/202530167-fb5ce442-847d-4c84-84db-28286300c97a.png)